src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -168,6 +177,51 @@ private static ScriptHostConfiguration GetScriptHostConfiguration(string scriptP
             return scriptHostConfig;
         }
 
+        private static string MakeValidHostId(string id)
```

Comment:
This was logic added for the CLI changes which is where you (correctly) based your work, which is why is showing as an addition to the runtime. We should be able to use your implementation when this logic is moved to the CLI (depending on how @ahmelsayed chooses to approach it there)